src/pages/Start.styles.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components/macro";
+import { Link as RouterLink } from "@reach/router";
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 24px 24px;
+    max-width: 700px;
+    height: 100vh;
+    margin: 0 auto;
+`;
+
+const HypeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+`;
+
+const HypeSubText = styled.p`
+    color: ${({theme}) => theme.COLOURS.accent};
+    margin: 32px 16px 16px 16px;
+`;
+
+const ReadyButton = styled.button`
+background-color: ${({theme}) => theme.COLOURS.accent};
```

Comment:
We should get some autoformatting stuff running somewhere, at some point. Probably not important at all right now tho haha.

Reply:
Definitely! I actually had to add prettier to this PR because otherwise 'toMatchInlineSnapshot' threw an error (https://github.com/facebook/jest/issues/8467)...